### DISCUSSION TITLE
نصب نشدن روی .net framework 4.6

### DISCUSSION THREAD

**Discussion by sinath72** [Sina-Soltani/Parbad#196] [Q&A] [answered]
سلام
من میخوام از طریق nuget پکیج شما رو نصب کنم نمیشه و اروور میده

![err](https://user-images.githubusercontent.com/56226543/113562535-81268200-961b-11eb-8c22-edc45c18c547.PNG)

لطفا ای پکیج رو برای 4.6 هم سازگار بشه ممنون

**Comment by Sina-Soltani** [comment, marked as answer]
درود

امکان پذیر نیست

از اونجایی که این کتابخونه برای تمام محصولات وب مایکروسافت تهیه شده، بر پایه
.NetStandard2.0
و بالاتر درست شده

تعدادی از امکانات و تکنولوژی های استفاده شده در این پکیج در نسخه قبل از دو موجود نیستند

طبق [مستندات](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) مایکروسافت، نسخه ای که شما مد نظر دارید، سازگار با
.NetStandard2.0
نیست

بنابراین تنها راه موجود برای شما اینه که پروژه رو به
.NetFramework4.6.2
و یا بالاتر ارتقا بدید تا بتونه با
.NetStandard2.0
هماهنگ باشه
